### PR TITLE
feat (charges): change instant to pay_in_advance

### PR DESCRIPTION
**Pull request by lovrocolic**

### CODE REVIEW

File: docs/api/03_plans/create-plan.mdx [hunk 1]
```diff
@@ -374,7 +374,7 @@ curl --location --request POST "$LAGO_URL/api/v1/plans" \
     createdAt: null,
     lagoBillableMetricId: "id",
     chargeModel: "graduated" as ChargeObject["chargeModel"],
-    instant: false,
+    pay_in_advance: false,
```
Comment (vincent-pochet) [inline docs/api/03_plans/create-plan.mdx:377]:
```suggestion
    payInAdvance: false,
```

File: docs/api/03_plans/create-plan.mdx [hunk 2]
```diff
@@ -398,7 +398,7 @@ curl --location --request POST "$LAGO_URL/api/v1/plans" \
     createdAt: null,
     lagoBillableMetricId: "id",
     chargeModel: "package" as ChargeObject["chargeModel"],
-    instant: false,
+    pay_in_advance: false,
```
Comment (vincent-pochet) [inline docs/api/03_plans/create-plan.mdx:401]:
```suggestion
    payInAdvance: false,
```

File: docs/api/03_plans/create-plan.mdx [hunk 3]
```diff
@@ -413,7 +413,7 @@ curl --location --request POST "$LAGO_URL/api/v1/plans" \
     createdAt: null,
     lagoBillableMetricId: "id",
     chargeModel: "percentage" as ChargeObject["chargeModel"],
-    instant: false,
+    pay_in_advance: false,
```
Comment (vincent-pochet) [inline docs/api/03_plans/create-plan.mdx:416]:
```suggestion
    payInAdvance: false,
```

File: docs/api/03_plans/create-plan.mdx [hunk 4]
```diff
@@ -429,7 +429,7 @@ curl --location --request POST "$LAGO_URL/api/v1/plans" \
     createdAt: null,
     lagoBillableMetricId: "id",
     chargeModel: "volume" as ChargeObject["chargeModel"],
-    instant: false,
+    pay_in_advance: false,
```
Comment (vincent-pochet) [inline docs/api/03_plans/create-plan.mdx:432]:
```suggestion
    payInAdvance: false,
```

Comment (lovrocolic) [reply docs/api/03_plans/create-plan.mdx]: Thanks for catching 😅 Fixed in all places

**Pull request merged**